test/CMakeLists.txt

```diff
@@ -362,6 +362,24 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU
     )
 endif()
 
+# Only produce timing tests for UNIX based systems (faketime constraint)
+find_library(
+        LIBFAKETIME
+        NAMES faketime
+        PATHS "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/lib"
+        NO_DEFAULT_PATH
```

Comment:
```suggestion
```

Same. CMake's automatic logic should find libfaketime and prefer the binaries from vcpkg over the binaries from the system.

Reply:
Removed